src/local-ssh/server.ts

```diff
@@ -59,6 +67,28 @@ export class LocalSSHGatewayServer {
 		return session;
 	}
 
+	private async tryReloadDaemon() {
+		if (this.clientCount > 0) {
```

Comment:
Does it mean we would run several daemons in parallel? i.e. if some window already upgraded and assumes a never version of daemon, but others are not yet?

Reply:
It's connection count (need to rename it)